docs/src/components/Guidelines/index.tsx

```diff
@@ -0,0 +1,168 @@
+import { slugify } from "../../utils/common";
+
+export interface GuidelineType {
+  type: "do" | "dont";
+}
+
+interface GuidelineProps extends GuidelineType {
+  title: string;
+  children: React.ReactNode;
+}
+
+const StyledComponent = styled.div`
+  ${({ theme }) => css`
+    background-color: ${theme.orbit.paletteCloudLight};
+    border-radius: ${theme.orbit.spaceMedium};
+    padding: ${theme.orbit.spaceMedium} 0;
+  `}
+`;
+
+interface WrapperProps extends GuidelineType {
+  border: boolean;
+}
+
+const Wrapper = styled.div<WrapperProps>`
+  ${({ border, theme, type }) => css`
+    ${border &&
+    `border-left: 4px solid ${
+      type === "do" ? theme.orbit.paletteGreenNormal : theme.orbit.paletteRedNormal
+    }`};
+    padding: ${theme.orbit.spaceXSmall} ${theme.orbit.spaceLarge} ${theme.orbit.spaceXSmall}
+      ${theme.orbit.spaceMedium};
+    display: flex;
+  `}
+`;
+
+const ContentContainer = styled.div`
+  ${({ theme }) => css`
+    padding-left: ${theme.orbit.spaceXSmall};
+  `}
+`;
+
+interface ImageContainerProps {
+  noLeftPadding?: boolean;
+  middleAlign?: boolean;
+  isMediumMobile?: boolean | null;
+}
+
+const ImageContainer = styled.div<ImageContainerProps>`
+  ${({ isMediumMobile, noLeftPadding, theme }) => css`
+    padding: ${noLeftPadding ? theme.orbit.spaceMedium : theme.orbit.spaceXLarge};
+    ${noLeftPadding && "padding-left: 0;"}
+    width: ${isMediumMobile ? "346px" : "100%"};
+    background-color: ${theme.orbit.paletteWhite};
+    border-radius: ${theme.orbit.spaceMedium};
+  `}
+`;
+
+const ImageBorder = styled.div<GuidelineType>`
+  ${({ theme, type }) => css`
+    border-left: 4px solid
+      ${type === "do" ? theme.orbit.paletteGreenNormal : theme.orbit.paletteRedNormal};
+    padding: ${theme.orbit.spaceMedium};
+    padding-left: ${theme.orbit.spaceXLarge};
+    height: 100%;
+  `}
+`;
+
+const DoDontHeaderWrapper = styled.div`
+  ${({ theme }) => css`
+    padding-left: ${theme.orbit.spaceLarge};
+  `}
+`;
+
+export const DoDontHeader = ({ type }: GuidelineType) => (
+  <Text weight="bold" uppercase type={type === "do" ? "success" : "critical"}>
+    {type === "do" ? "Do" : "Don't"}
+  </Text>
+);
+
+export default function Guideline({ type = "do", title, children }: GuidelineProps) {
+  const { isDesktop, isMediumMobile, isTablet } = useMediaQuery();
+
+  const images: React.ReactNode[] = [];
+
+  const checkIfImageAndAddToArray = object => {
+    if (object.props?.children?.props?.className === imageWrapperClass) {
+      images.push(object);
+      return true;
+    }
+    return false;
+  };
+
+  const extractContent = object => {
+    if (Array.isArray(object)) {
+      return React.Children.map(object, child => {
+        if (checkIfImageAndAddToArray(child)) {
+          return "";
+        }
+        return child;
+      });
+    }
+    if (checkIfImageAndAddToArray(object)) {
+      return ""; // only should apply for guidelines with only an image
+    }
+    return object;
+  };
```

Comment:
I am not happy with this check and functions, looks like it can be done more simple, I'll check it more right now and will put a suggestion here :) 

something like that can be done: 
```
  const isImage = object => {
    if (object.props?.children?.props?.className === imageWrapperClass) return true;
    return false;
  };

  interface Content {
    images: React.ReactNode[];
    content: React.ReactNode;
  }

  const getContent = data => {
    return React.Children.toArray(data).reduce(
      (acc: Content, cur) => {
        if (isImage(cur)) acc.images.push(cur);
        else acc.content = cur;

        return acc;
      },
      { images: [], content: null },
    );
  };

  const { images, content } = getContent(children);
```